File: mefApi/Models/Utilisateur.cs
using mefapi.Enums;
using System.ComponentModel.DataAnnotations;

namespace mefApi.Models
{
    public class Utilisateur : BaseEntity
    {
        public string? Login { get; set; }
        public byte[]? Password { get; set; }
        public byte[]? PasswordKey { get; set; }
        public Role? Role { get; set; } 
        public string? Photo { get; set; }
    }
}

File: mef/src/app/core/models/utilisateur.ts
import { Role } from './role';

export class Utilisateur {
  id: number;
  login: string;
  pasword: string;
  role: Role;
  photo: string;

  constructor(
    id: number,
    login: string,
    password: string,
    role: Role,
    photo: string
  ) {
    this.id = id;
    this.login = login;
    this.pasword = password;
    this.role = role;
    this.photo = photo;
  }
}
